src/NuGet.Services.AzureSearch/ScoringProfiles/NewlyPublishedAndDownloadCountBoosterProfile.cs

```diff
@@ -1,33 +1,45 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
 using System.Collections.Generic;
 using Microsoft.Azure.Search.Models;
 using NuGet.Services.AzureSearch.SearchService;
 
 namespace NuGet.Services.AzureSearch.ScoringProfiles
 {
-    public class DownloadCountBoosterProfile
+    public class NewlyPublishedAndDownloadCountBoosterProfile
     {
-        public const string Name = "download_count_scoring_profile";
+        public const string Name = "newly_published_and_download_count_scoring_profile";
 
         public static readonly ScoringProfile Instance = new ScoringProfile(
             Name,
             textWeights: new TextWeights
             {
                 Weights = new Dictionary<string, double>
                 {
-                    { IndexFields.PackageId, 10 },              // Exact match of the package id should be boosted higher.
-                    { IndexFields.TokenizedPackageId, 5 }
+                    { IndexFields.PackageId, 10 },              // Exact match of the package id should be boosted the highest.
+                    { IndexFields.TokenizedPackageId, 5 },
+                    { IndexFields.Tags, 1 },
```

Comment:
I believe @shishirx34 set this to be explicit, as it seems that a weight of `1` has no effect.  I will follow up with Azure Search team on this.